conferences/PyData_Global_2021/tutorial_proposal.md

```diff
@@ -0,0 +1,16 @@
+# PyData2021 tutorial: "How to improve your time series forecast using sktime"
+
+
+## Short description
+Time series are ubiquitous in real-world applications, but often add considerable complications to data science workflows. In this tutorial, you’ll learn about how to apply and extend machine learning techniques to time series. The tutorial will continue the [PyData tutorial from 2020](https://www.youtube.com/watch?v=Wf2naBHRo8Q&t=1240s) with a deeper dive into pipelining, tuning, backtesting. Further, you will learn about the learning problems of time series annotation and time series clustering.
```

Comment:
hm, not sure whether this is the right framing - this sounds like it´s incremental on the 2020 proposal. Also note that 2020 was just local pydata (Netherlands I think).

I would frame it as:

* first pydata global introduction to `sktime`; `sktime` interface has considerably matured since 2020, now it's the new "consolidated" one; tutorials have been refactored and have matured
* introducing new taxonomy of learning tasks and related features: tasks, tags, datatypes, scitypes (?)
* "how to identify your learning task"
* new & exciting functionality for composition, evaluation
* new & exciting feature for extenders: extension templates, developer tutorial

pydata 2020 should be cited, but this tutorial should not be "v2"

Reply:
In my view, scitypes, datatypes and tags go beyond what one can cover in a 90-min introductory session.